barretenberg/cpp/src/barretenberg/commitment_schemes/claim.hpp

```diff
@@ -51,6 +53,20 @@ template <typename Curve> class OpeningClaim {
     // commitment to univariate polynomial p(X)
     Commitment commitment;
 
+    IPAClaimIndices get_witness_indices() const
+        requires(std::is_same_v<Curve, stdlib::grumpkin<UltraCircuitBuilder>>)
+    {
+        return { opening_pair.challenge.binary_basis_limbs[0].element.normalize().witness_index,
+                 opening_pair.challenge.binary_basis_limbs[1].element.normalize().witness_index,
+                 opening_pair.challenge.binary_basis_limbs[2].element.normalize().witness_index,
+                 opening_pair.challenge.binary_basis_limbs[3].element.normalize().witness_index,
+                 //  opening_pair.evaluation.binary_basis_limbs[0].element.normalize().witness_index,
```

Comment:
the evaluation is always 0 for our case I believe, so it doesn't have any witness indices

Reply:
actually, when we accumulate, the evaluation will not be 0, so the correct thing here is to include these

Reply:
I'll do it in the followup PR because its not needed until we actually do accumulation